app/renderer/components/tooltip.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const TaskStatus = require('../../common/task_status');
+
+module.exports = {
+    props: ['task'],
+    template: `
+    <div class="tooltip">
+        <div class="tooltip-text">
+            {{ status }}
+        </div>
+    </div>
+    `,
+    computed: {
+        status() {
+            switch (this.task.status) {
+                case TaskStatus.idle:
+                case TaskStatus.stopped:
+                    return "Stopped";
```

Comment:
Wich is better as text, `stopped` or `idle`? (open to discussion)

Reply:
I feel like `stopped` is better since `idle` makes it sound like it's suspended.